vespa/metadata.csv

```diff
@@ -0,0 +1,48 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+vespa.http.status.1xx.rate,gauge,,response,second,Number of responses with a 1xx status,0,vespa,
+vespa.http.status.2xx.rate,gauge,,response,second,Number of responses with a 2xx status,0,vespa,
+vespa.http.status.3xx.rate,gauge,,response,second,Number of responses with a 3xx status,0,vespa,
+vespa.http.status.4xx.rate,gauge,,response,second,Number of responses with a 4xx status,0,vespa,
+vespa.http.status.5xx.rate,gauge,,response,second,Number of responses with a 5xx status,0,vespa,
+vespa.jdisc.gc.ms.average,gauge,,millisecond,,Time spent in GC,0,vespa,
+vespa.mem.heap.free.average,gauge,,byte,,Free heap size,0,vespa,
+vespa.queries.rate,gauge,,query,second,Number of search queries,1,vespa,
+vespa.feed.operations.rate,gauge,,operation,second,Number of feed operations,1,vespa,
+vespa.query_latency.average,gauge,,millisecond,,Query processing time,-1,vespa,
+vespa.query_latency.95percentile,gauge,,millisecond,,95 percentile query processing time,-1,vespa,
+vespa.query_latency.99percentile,gauge,,millisecond,,99 percentile query processing time,-1,vespa,
+vespa.hits_per_query.average,gauge,,document,query,"Hits in the returned result, per query",0,vespa,
+vespa.totalhits_per_query.average,gauge,,hit,query,Estimated total number of hits per query,0,vespa,
+vespa.degraded_queries.rate,gauge,,query,second,Queries with degraded results due to timeout,-1,vespa,
+vespa.failed_queries.rate,gauge,,query,second,Failed queries,-1,vespa,
+vespa.serverActiveThreads.average,gauge,,thread,,Threads that are active processing requests,0,vespa,
+vespa.content.proton.docsum.docs.rate,gauge,,document,,Total docsums returned,0,vespa,
```

Comment:
Could we map the `vespa.content.proton...` metrics to a shorter version? In this example `vespa.proton.docsum.total` might better express the purpose of the metric.

Reply:
From this PR alone, it may look as if `content` is redundant. However, Vespa has a lot more metrics than those currently listed in `metadata.csv`.  For example, we have a family of metrics with the prefix `vespa.searchnode.proton` and others. It could well be that we add some of those metrics later, so to avoid confusion we would like to keep the current names.